launcher/ui/pages/global/APIPage.ui

```diff
@@ -147,6 +147,58 @@
          </layout>
         </widget>
        </item>
+      <item>
+        <widget class="QGroupBox" name="groupBox_cfproxy">
+         <property name="title">
+          <string>&amp;Curseforge Proxy</string>
+         </property>
+         <layout class="QVBoxLayout" name="verticalLayout_41">
+          <item>
+           <widget class="Line" name="line_21">
+            <property name="orientation">
+             <enum>Qt::Horizontal</enum>
+            </property>
+           </widget>
+          </item>
+          <item>
+           <widget class="QLabel" name="label_31">
+            <property name="text">
+             <string>Note: you probably don't need to set this if Curseforge mod downloading already works.</string>
```

Comment:
Maybe add here that using the CurseForge API directly is unsupported, as we currently don't have a way to set an API key here